ts/sagas/wallet.ts

```diff
@@ -343,6 +345,62 @@ function* showWalletOrSelectPsp(idWallet: number, paymentId?: string) {
   }
 }
 
+const MAX_RETRIES_POLLING = 20;
+const DELAY_BETWEEN_RETRIES_MS = 1000;
+
+const delay = (ms: number): Promise<undefined> =>
+  new Promise(resolve => setTimeout(resolve, ms));
+
+const pollForPaymentId = async (
```

Comment:
you can use the existing retry logic we use for the backend client by customizing it, I'll send you a PR

Reply:
https://github.com/teamdigitale/italia-app/pull/383/files#diff-bc352e9e1898a6991158294074f1de7dR386